src/olympia/blocklist/models.py

```diff
@@ -155,14 +156,24 @@ def get_blocks_from_guids(cls, guids):
         return blocks
 
 
+class BlockType(Enum):
+    BLOCKED = 'blocked'
+    SOFT_BLOCKED = 'soft_blocked'
+
 class BlockVersion(ModelBase):
     version = models.OneToOneField(Version, on_delete=models.CASCADE)
     block = models.ForeignKey(Block, on_delete=models.CASCADE)
     soft = models.BooleanField(default=False)
 
     def __str__(self) -> str:
-        blocktype = 'soft' if self.soft else 'hard'
-        return f'Block.id={self.block_id} ({blocktype}) -> Version.id={self.version_id}'
+        return (
+            f'Block.id={self.block_id} ({self.block_type}) '
+            f'-> Version.id={self.version_id}'
+        )
+
+    @property
+    def block_type(self):
+        return BlockType.SOFT_BLOCKED if self.soft else BlockType.BLOCKED
```

Comment:
with a Choices object you can get the value from self.soft.display or similar, you don't need a `block_type`

Reply:
any field is a choice field in django 😉- it's the use of Choices to declare the constants that's important.